src/Main.elm

```diff
@@ -290,6 +286,54 @@ titleField model =
         )
 
 
+bodyField : Model -> Html Msg
+bodyField model =
+  let
+    body =
+      Form.getFieldAsString "body" model.form
+
+    conditionalProperties =
+      case body.liveError of
+        Just error ->
+          case error of
+            Form.Error.InvalidString ->
+              [ Textfield.error "Can't be blank" ]
+
+            Form.Error.Empty ->
+              [ Textfield.error "Can't be blank" ]
+
+            _ ->
+              [ Textfield.error <| toString error ]
+
+        Nothing ->
+          []
+  in
+      Textfield.render Mdl
+        [ 1, 0 ]
+        model.mdl
+        ([ Textfield.label "Body"
+         , Textfield.floatingLabel
+         , Textfield.text'
```

Comment:
`Textfield.text'` is single line input.
`Textfield.textarea` is multiline input, is probably what you want for a proposal body.
Use `Textfield.row n` to specify number of rows.